src/components/common/checkbox/page.jsx

```diff
@@ -21,9 +20,4 @@ export default function CheckBox({ label, value, onChange, name }) {
   );
 }
 
-CheckBox.propTypes = {
```

Comment:
you remove the PropTypes code but it seems like it give error of types validation so please handle that as well